modules/installation-configuration-parameters.adoc

```diff
@@ -452,6 +532,82 @@ link:https://yaml.org/spec/1.2/spec.html#sequence//[YAML sequence].
 
 endif::gcp[]
 
+ifdef::rhv[]
+
+.Additional {rh-virtualization-first} parameters for clusters
+[cols=".^2,.^3a,.^3a",options="header"]
+|====
+|Parameter|Description|Values
+
+|`platform.ovirt.ovirt_cluster_id`
+|Required. The Cluster where the VMs will be created.
+|String. For example: `68833f9f-e89c-4891-b768-e2ba0815b76b`
+
+|`platform.ovirt.ovirt_storage_domain_id`
+|Required. The Storage Domain ID where the VM disks will be created.
+|String. For example: `ed7b0f4e-0e96-492a-8fff-279213ee1468`
+
+|`platform.ovirt.ovirt_network_name`
+|Required. The network name where the VM nics will be created.
+|String. For example: `ocpcluster`
+
+|`platform.ovirt.vnicProfileID`
+|Required. The vNIC profile ID of the VM network interfaces. This can be inferred if the cluster network has a single profile.
+|String. For example: `3fa86930-0be5-4052-b667-b79f0a729692`
+
+|`platform.ovirt.api_vip`
+|Required. An IP address on the machineNetwork that will be assigned to the API VIP.
+|String. Example: `10.46.8.230`
+
+|`platform.ovirt.ingress_vip`
+|Required. An IP address on the machineNetwork that will be assigned to the Ingress VIP.
+|String. Example: `10.46.8.232`
+|====
+
+
+.Additional {rh-virtualization} parameters for machine pools
+[cols=".^2,.^3a,.^3a",options="header"]
+|====
+|Parameter|Description|Values
+
+|`<machine-pool>.platform.ovirt.cpu`
+|Optional. Defines the CPU of the VM.
+|Object
+
+|`<machine-pool>.platform.ovirt.cpu.cores`
+|Required if you use `<machine-pool>.platform.ovirt.cpu`. The number of cores. Total vCPUs is cores * sockets.
+|Integer
+
+|`<machine-pool>.platform.ovirt.cpu.sockets`
+|Required if you use `<machine-pool>.platform.ovirt.cpu`. The number of sockets. Total vCPUs is cores * sockets.
+|Integer
+
+|`<machine-pool>.platform.ovirt.memoryMB`
+|Optional. Memory of the VM in MiB.
+| Integer
+
+|`<machine-pool>.platform.ovirt.instanceTypeID`
+|Optional. The VM instance-type, such as `tiny`, `small`, `medium`, `large`, or `xlarge`.
```

Comment:
I'm not totally sure about this. However, it's been a long time since I've tested that, so I'm really not sure. Can we confirm with Roy that this actually takes strings like "tiny" or "large"? Or is the actual ID (e.g. 00000009-0009-0009-0009-0000000000f1) as found at https://engine-fqdn/ovirt-engine/api/instancetypes required? If you've already confirmed that, feel free to ignore this comment.

Reply:
@rgolangh Please comment on this question.

Reply:
@jan-zmeskal This is the only open question. I was waiting for Roy to comment. I'll publish it as-is and update at needed.

Reply:
@rolfedh I agree. I'm not able to confirm it at this moment, so let's go for it.

Reply:
The value should be a string in UUID. You are right that its misleading to mention the name of those instance type4 like "small", "tiny", and so on, without mentioning that the value must be the UUID of those entities.